tests/berlin/eip2930_access_list/test_tx_intrinsic_gas.py

```diff
@@ -0,0 +1,198 @@
+        pytest.param(
+            [
+                AccessList(
+                    address=Address(1),
+                    storage_keys=[
+                        Hash("0x60A7", left_padding=True),
+                        Hash("0x60A8", left_padding=True),
+                    ],
+                )
+            ],
+            id="access_list_1_address_2_keys",
+        ),
+        pytest.param(
+            [
+                AccessList(address=Address(1), storage_keys=[]),
+                AccessList(address=Address(2), storage_keys=[]),
+            ],
+            id="access_list_2_address_empty_keys",
+        ),
+        pytest.param(
+            [
+                AccessList(address=Address(1), storage_keys=[]),
+                AccessList(address=Address(2), storage_keys=[Hash("0x60A7", left_padding=True)]),
+            ],
+            id="access_list_2_address_1_keys",
+        ),
+        pytest.param(
+            [
+                AccessList(address=Address(1), storage_keys=[Hash("0x60A7", left_padding=True)]),
+                AccessList(address=Address(2), storage_keys=[Hash("0x60A8", left_padding=True)]),
+            ],
+            id="access_list_2_address_2_keys",
+        ),
+        pytest.param(
+            [
+                AccessList(
+                    address=Address(1),
+                    storage_keys=[
+                        Hash("0x60A7", left_padding=True),
+                        Hash("0x60A8", left_padding=True),
+                    ],
+                ),
+                AccessList(address=Address(2), storage_keys=[]),
+            ],
+            id="access_list_2_address_2_keys_inversion",
+        ),
+        pytest.param(
+            [
+                AccessList(
+                    address=Address(1),
+                    storage_keys=[
+                        Hash("0xce11", left_padding=True),
+                    ],
+                ),
+                AccessList(
+                    address=Address(2),
+                    storage_keys=[
+                        Hash("0x60A7", left_padding=True),
+                    ],
+                ),
+                *[
+                    AccessList(
+                        address=Address(i),
+                        storage_keys=[
+                            Hash("0x600D", left_padding=True),
+                            Hash("0x0BAD", left_padding=True),
+                            Hash("0x60A7", left_padding=True),
+                            Hash("0xBEEF", left_padding=True),
+                        ],
+                    )
+                    for i in range(3, 13)  # 3 to 12 inclusive (10 entries)
+                ],
+            ],
+            id="access_list_12_address_42_keys",
+        ),
+    ],
+)
+@pytest.mark.parametrize(
+    "below_intrinsic",
+    [
+        pytest.param(False),
+        pytest.param(True, marks=pytest.mark.exception_test),
+    ],
+)
+def test_tx_intrinsic_gas(
+    state_test: StateTestFiller,
```

Comment:
```suggestion
    state_test: StateTestFiller,
    fork: Fork,
```
If the previous comment is applied.